apps/myaccount/webpack.config.js

```diff
@@ -110,7 +110,16 @@ module.exports = (env) => {
                             loader: "@svgr/webpack",
                             options: {
                                 svgoConfig: {
-                                    plugins: [{ prefixIds: false }]
+                                    plugins: [{
```

Comment:
Since this caused weird a behaviour in icons, lets remove this.

Reply:
Fixed with efe4b13